python/lsst/pipe/tasks/ingestCalibs.py

```diff
@@ -198,6 +225,10 @@ def run(self, args):
                     self.log.warn(str("Skipped adding %s of observation type '%s' to registry" %
                                       (infile, calibType)))
                     continue
+                outfile = self.parse.getDestination(args.butler, fileInfo, infile)
```

Comment:
This information is printed out as part of the IngestTask, but I can change it to a log info entry.